### PR TITLE
Subregion-toggle

### PR DESCRIPTION
**Pull request by jflaboe**

### CODE REVIEW

File: site/map.js [hunk 1]
```diff
@@ -38,35 +38,52 @@ function initData() {
...
     type: 'FeatureCollection',
     features: data.features.features.filter(function(feature) {
       const location = data.locations[feature.properties.locationId];
-      if (location && !location.county && location.aggregate === 'county') {
-        return false;
+      if (showSubregion) {
+        if (location && !location.county && location.aggregate === 'county') {
+          return false;
+        }
       }
```
Comment (camjc) [inline site/map.js]:
```suggestion
      if (showSubregion && location && !location.county && location.aggregate === 'county') {
        return false;
      }
```

File: site/map.js [hunk 2]
```diff
@@ -38,35 +38,52 @@ function initData() {
...
   const countryFeatures = {
     type: 'FeatureCollection',
     features: data.features.features.filter(function(feature) {
       const location = data.locations[feature.properties.locationId];
-      if (location && location.country && !location.state && location.aggregate === 'state') {
-        return false;
+      if (showSubregion) {
+        if (location && location.country && !location.state && location.aggregate === 'state') {
+          return false;
+        }
```
Comment (camjc) [inline site/map.js]: Feels repetitious of the generateStateFeatures, should/can we abstract?

**Comment by camjc** [comment]
Looking cool 👍

**Comment by praging** [comment]
See https://github.com/covidatlas/coronadatascraper/issues/616 for a possible issue.

**Comment by jzohrab** [comment]
@lazd , should this get merged into the mapupdates branch (last commits of this were from about 2 weeks ago : https://github.com/covidatlas/coronadatascraper/commits/mapupdates )?

This PR is going into that feature branch, and not into master.  There are some questions on it, but it looks like it could keep that branch moving forward.  Thoughts?

**Comment by lazd** [comment]
@jzohrab not sure, but we should look at this and #367 and decide if we want those features on our map, or if we should move that development to the COVIDAtlas frontend work?

**Comment by lazd** [comment]
Refactored this a bit and merges this into mapupdates, thanks!

**Pull request merged**